.travis.yaml

```diff
@@ -8,3 +8,4 @@ script:
   - tox
 notifications:
 email: rpappalax@gmail.com 
```

Comment:
probably want to change this either to your own email (at first), remove or add fte-ci email if you wish to track the travis runs